docs/FAQ.md

```diff
@@ -0,0 +1,45 @@
+# **<center>FAQ'S</center>**
+
+**Q1.** Who all can use this website?
```

Comment:
Can make all the question bold? Also, make their font larger than the answer

Reply:
I can make it bold , and will try to make font larger but at that time I tried it wasn't happening

Reply:
if use # it becomes a heading so, try using a heading with the smallest size. I know ### is small but not sure if #### exists

Reply:
Hey can u explain this again heading refers to faq or what , I have done all the other changes so once this one is done too then I will make pr